builder/bin/publish-release-controller.go

```diff
@@ -77,24 +77,26 @@ func main() {
 
 	res, err := client.Do(req)
 
+	// Read json response from body
+	body, _ := ioutil.ReadAll(res.Body)
```

Comment:
We're ignoring a possible `error` return from `ioutil.ReadAll` here, then reporting any error that `client.Do` may have raised. I think we should handle `client.Do`'s error immediately afterward, and then have similar error handling for `ioutil.ReadAll`. I realize this isn't your fault, the error handling here was a bit disjointed already, and this is one of the uglier aspects of go coding, but I think explicit handling is better here.